package.json

```
@@ -8,13 +8,17 @@
   },
   "scripts": {
     "install": "node ./download-chromedriver.js",
+    "lint": "standard --fix",
     "test": "standard && mocha"
   },
+  "engines" : { 
+    "node" : ">=10.0.0" 
```

Actually needs Node 10.12 because I got rid of the `mkdirp` dependency ✂️ 

```suggestion
    "node" : ">=10.12.0" 
```